package.json

```diff
@@ -1,70 +1,91 @@
-    "tslib": "^2.3.1",
-    "zone.js": "~0.11.5"
-  },
-  "devDependencies": {
-    "@angular-devkit/build-angular": "^15.0.0",
-    "@angular-eslint/builder": "14.1.2",
-    "@angular-eslint/eslint-plugin": "14.1.2",
-    "@angular-eslint/eslint-plugin-template": "14.1.2",
-    "@angular-eslint/schematics": "^14.1.2",
-    "@angular-eslint/template-parser": "14.1.2",
-    "@angular/cli": "^15.0.0",
-    "@angular/compiler-cli": "^15.0.0",
-    "@angular/language-service": "^15.0.0",
-    "@semantic-release/changelog": "^6.0.1",
-    "@stackblitz/sdk": "^1.6.0",
-    "@types/jasmine": "~4.0.3",
-    "@types/jasminewd2": "^2.0.10",
-    "@types/node": "^17.0.35",
-    "@typescript-eslint/eslint-plugin": "5.26.0",
-    "@typescript-eslint/parser": "5.26.0",
-    "coveralls": "^3.1.1",
-    "eslint": "8.16.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsdoc": "39.3.2",
-    "eslint-plugin-prefer-arrow": "^1.2.2",
-    "gh-pages-travis": "^1.0.4",
-    "jasmine-core": "~4.1.1",
-    "jasmine-spec-reporter": "~7.0.0",
-    "karma": "~6.3.20",
-    "karma-chrome-launcher": "~3.1.1",
-    "karma-coverage": "^2.2.0",
-    "karma-jasmine": "~5.0.1",
-    "karma-jasmine-html-reporter": "^2.0.0",
-    "ng-packagr": "^15.0.0",
-    "node-sass": "^7.0.1",
-    "semantic-release": "^19.0.2",
-    "standard-version": "^9.5.0",
-    "ts-node": "~10.8.0",
-    "typescript": "~4.8.4"
-  }
-}
+{
+  "repository": "@commonapp/ng-select",
+  "engines": {
+    "node": ">= 16.20.0",
+    "npm": ">= 8.19.4"
+  },
+  "scripts": {
+    "build": "ng build ng-select && ng build ng-option-highlight && yarn build:themes && yarn copy-sass",
+    "build:demo": "ng build demo --configuration production --base-href=/ng-select && yarn copy-examples",
+    "build:themes": "node-sass --output-style compressed src/ng-select/themes/ -o dist/ng-select/themes",
+    "copy-sass": "mkdir -p dist/ng-select/scss && cp src/ng-select/**/*.scss dist/ng-select/scss",
+    "copy-examples": "cp -r src/demo/app/examples dist/demo",
+    "start": "ng serve",
+    "test:watch": "ng test ng-select --watch",
+    "test": "ng test ng-select --code-coverage && ng test ng-option-highlight --code-coverage",
+    "test:ci": "ng test --watch=false --browsers=ChromeHeadless --code-coverage",
+    "coveralls": "cat ./coverage/ng-select/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "lint": "ng lint ng-select && ng lint ng-option-highlight",
+    "postpublish": "cp dist/ng-select/package.json src/ng-select && cp dist/ng-option-highlight/package.json src/ng-option-highlight",
+    "release": "semantic-release"
+  },
+  "release": {
+    "branch": "master",
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/changelog",
+      "@semantic-release/github",
+      [
+        "@semantic-release/npm",
+        {
+          "pkgRoot": "dist/ng-select"
+        }
+      ],
+      "@semantic-release/release-notes-generator"
+    ]
+  },

```

Comment:
We can nix this as well as we don't use their automated release strategy.

Reply:
@geebru done.